opm/autodiff/BlackoilModelBase_impl.hpp

```diff
@@ -2290,6 +2290,73 @@ namespace detail {
 
 
 
+
+    template <class Grid, class WellModel, class Implementation>
+    std::vector<V>
+    BlackoilModelBase<Grid, WellModel, Implementation>::
+    computeFluidInPlace(const ReservoirState& x,
+                        const WellState& xw,
+                        const std::vector<int>& fipnum)
+    {
+        using namespace Opm::AutoDiffGrid;
+        const int nc = numCells(grid_);
+        const int np = x.numPhases();
+
+        SolutionState state(np);
```

Comment:
I was not planning on immediate refactoring, no. But if the need arises, I'd like to see it done rather than postponed. If you can do with the accum data, why was it not used here? (Also: why not accum[1]?)